src/notification/notification.component.ts

```diff
@@ -120,14 +120,14 @@ export class Notification {
 		this.close.emit();
 	}
 
-	onClick(action) {
+	onClick(action, event) {
 		if (!action) {
 			return;
 		}
 		if (isObservable(action)) {
-			(action as Subject<any>).next();
+			(action as Subject<{event: Event, action: any}>).next({event, action: event.action});
```

Comment:
I'm not sure that `event.action` is the right `action` ... instead of passing `action.click` in the template, you'll want to pass the whole `action` object and just call `.click` in here :+1: